tests/upgrades/test_classparameter.py

```diff
@@ -130,9 +130,10 @@ def test_pre_puppet_class_parameter_data_and_type(
         self._validate_value(data, sc_param)
 
     @pytest.mark.post_upgrade(depend_on=test_pre_puppet_class_parameter_data_and_type)
+    @pytest.mark.usefixtures('_clean_scenario')
     @pytest.mark.parametrize('count', list(range(1, 10)))
     def test_post_puppet_class_parameter_data_and_type(
-        self, count, _clean_scenario, class_pre_upgrade_data, class_target_sat
+        self, count, class_pre_upgrade_data, class_target_sat
```

Comment:
Just wondering, if this change to use usefixtures here made by ruff ?

Reply:
this was a manual change based on one of the new rules ruff enforces.